CHANGELOG.md

```diff
@@ -1,125 +1,215 @@
 ## Changelog
+
+### v1.3.20
+
+#### Updated
+
+- Updated form.scss so that .alert-loading appears in the same way as
```

Comment:
Can you include that this is for the `Form` component? It's implied because of the scss file name but it'd be good to point it out. Also, can you add backticks to the file names and css classes? So 
```
`alert-loading/`
```